Assets/HoloToolkit/Input/Tests/Scripts/SphereGlobalKeywords.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.InputModule.Tests
+{
+    public class SphereGlobalKeywords : MonoBehaviour, ISpeechHandler
+    {
+        public void Start()
+        {
+            InputManager.Instance.AddGlobalListener(gameObject);
+        }
+
+        public void OnPhraseRecognized(PhraseRecognizedEventData eventData)
+        {
+            switch (eventData.Text.ToLower())
```

Comment:
I used the existing SphereKeyword.cs as a reference and it already uses a switch.